app/src/display/status_screen.c

```diff
@@ -65,5 +72,10 @@ lv_obj_t *zmk_display_status_screen() {
     zmk_widget_wpm_status_init(&wpm_status_widget, screen);
     lv_obj_align(zmk_widget_wpm_status_obj(&wpm_status_widget), LV_ALIGN_BOTTOM_RIGHT, 0, 0);
 #endif
+
+#if IS_ENABLED(CONFIG_ZMK_WIDGET_INDICATORS_STATUS)
+zmk_widget_hid_indicators_status_init(&indicators_status_widget, screen);
+    lv_obj_align(zmk_widget_hid_indicators_status_obj(&indicators_status_widget), LV_ALIGN_TOP_MID, 0, 0);
```

Comment:
On cheap 128x32 pixel ali express displays the arrow, when charging, is overlapping with the indicators, at least when reaching 100% (I cherry picked your commit and removed the spaces in between the letters, so it might be worst with your current layout). I moved the indicators to the bottom right - but I also have WPM disabled - and I, personally, like them there more.